src/lib/gas-refund.ts

```diff
@@ -19,6 +19,44 @@ export const GasRefundSafetyModuleStartEpoch = 11;
 export const GasRefundDeduplicationStartEpoch = 12;
 export const GasRefundTxOriginCheckStartEpoch = 12;
 export const GasRefundSPSPStakesAlgoFlipEpoch = 12;
+export const GasRefundConsiderContractTXsStartEpoch = 12;
+// todo: set this number accordingly
+export const GasRefundSwapSourceCovalentStartEpoch = 13;
+
+export const AUGUSTUS_ADDRESS = '0xDEF171Fe48CF0115B1d80b88dc8eAB59176FEe57'
```

Comment:
Augustus address isn't the same on all networks.

I suggest again to shoot all the code related to fetch swaps via covalent for now